src/components.d.ts

```diff
@@ -0,0 +1,9 @@
+// Ref: https://github.com/vuejs/core/pull/3399
+declare module 'vue' {
+  export interface GlobalComponents {
+    RouterLink: typeof import('vue-router')['RouterLink']
+    RouterView: typeof import('vue-router')['RouterView']
+  }
+}
+
+export {}
```

Comment:
In VS Code I use extension Volar, which recommends the addition of this file (and the deletion of the the other d.ts files below).